src/html/layout.html

```diff
@@ -55,7 +55,11 @@ <h1 class="logo-holder">
     </div>
   </main>
 
-  <footer class="footer"></footer>
+  <footer class="footer">
+    <div class="container">
+      {% include './components/social-share.html' %}
```

Comment:
Nie wiem co masz na myśli. Chcesz to wrzucić na górę strony?
W otwartym mobilnym menu chcę umieścić ten pasek na dole strony - będzie wtedy łatwo dostępny. Ale na desktopie nie wiem gdzie to wrzucić poza footerem.